third-party/realdds/include/realdds/dds-option.h

```diff
@@ -0,0 +1,53 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+
+#include <third-party/json_fwd.hpp>
+
+#include <string>
+
+
+namespace realdds {
+
+
+struct dds_option_range
+{
+    float min;
```

Comment:
I think we need to brainstorm this a bit more:
Options may not be numeric.
Options may not be float.
Options may not have ranges.
Options need to be very flexible.

I.e., I think we need a class hierarchy here.